frontend/tet/youth/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
There were some problem with next/core-web-vitals. You probably have to change it to `'plugin:@next/next/recommended',` when you adapt deepr to the frontend structure and start to use shared folder stuff.